docs/shared/node-server-tutorial/affected-project-graph.json

```diff
@@ -324,7 +322,7 @@
             "inputs": ["default", "{workspaceRoot}/.eslintrc.json"],
             "executor": "@nrwl/linter:eslint",
             "outputs": ["{options.outputFile}"],
-            "options": { "lintFilePatterns": ["./**/*.ts"] },
+            "options": { "lintFilePatterns": ["./src"] },
```

Comment:
The answer may come later in this PR, but how come this is a special case and still has a config?

Reply:
It's mentioned in the description of the PR:
> lint project generator will automatically set ./src to root/standalone projects unless lintFilePatterns is passed via options

Standard nested projects get `{projectRoot}` as a lint file pattern. This wouldn't work for root/standalone as it's root is `.` which would cause all files in the monorepo to be linter. That's why for those projects we use `./src` as a lint file pattern.